pyproject.toml

```diff
@@ -36,7 +36,7 @@ aiomisc = "^14.0.3"
 aiopg = "^1.2.1"
 aiohttp = "^3.7.4"
 dependency-injector = "^4.32.2"
-minos-microservice-common = "^0.2.0"
+minos-microservice-common = { git = "https://github.com/Clariteia/minos_microservice_common.git", branch = "issue-649-minos-config-services" }
```

Comment:
Do you really want to commit this?

Reply:
Hi @albamig, in this case the answer is yes, as it will be overridden by https://github.com/Clariteia/minos_microservice_networks/pull/444/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711L39 (I decided not to include that change on this PR to keep it simple) Thanks for the comment and well seen! 👀